src/bitcoin-cli.cpp

```diff
@@ -419,13 +422,13 @@ class NetinfoRequestHandler : public BaseRequestHandler
         // Report detailed peer connections list sorted by direction and minimum ping time.
         if (DetailsRequested() && !m_peers.empty()) {
             std::sort(m_peers.begin(), m_peers.end());
-            result += "Peer connections sorted by direction and min ping\n<-> relay   net  mping   ping send recv  txn  blk    age ";
+            result += strprintf("Peer connections sorted by direction and min ping\n<-> relay   net  mping   ping send recv  txn  blk %*s ", m_max_age_length, "age");
```

Comment:
43ee49f2c42f078763be55d666161507a2d7707b

`#include <tinyformat.h>` ?

Reply:
Well spotted. Will add it if I need to retouch or in the next PR to use the getpeerinfo connection types. There are quite a few `strprintf` in this file since years, so I guess it's not urgent.

Reply:
included

Reply:
Added the `tinyformat` header, and also `algorithm`, which was missing too. Thanks!